test/e2e/network_test.go

```diff
@@ -322,8 +361,9 @@ func (tc *testContext) createService(name string, serviceType v1.ServiceType, se
 			Type: serviceType,
 			Ports: []v1.ServicePort{
 				{
-					Protocol: v1.ProtocolTCP,
-					Port:     80,
+					Protocol:   v1.ProtocolTCP,
+					Port:       80,
```

Comment:
can we pass in the same targetport here?

Reply:
No, the targetport is the container's exposed port. `Port` is the port which is open on the Service. That should always be 80 for http.

Reply:
nit: code comment around this could be useful